staging/vhost-device-gpu/src/virt_gpu.rs

```diff
@@ -393,6 +394,58 @@ impl VirtioGpu {
         Ok(OkNoData)
     }
 
+    /// Updates the cursor's memory to the given resource_id, and sets its position to the given
+    /// coordinates.
+    pub fn update_cursor(
+        &mut self,
+        _resource_id: u32,
+        gpu_backend: &mut GpuBackend,
+        cursor_pos: VhostUserGpuCursorPos,
+        hot_x: u32,
+        hot_y: u32,
+        _mem: &GuestMemoryMmap,
+    ) -> VirtioGpuResult {
+        let size: usize = 4096;
+        let data = vec![0u32; size];
+        let mut data_array = [0u32; 4096];
+        data_array.copy_from_slice(&data[..]);
```

Comment:
This code doesn't seem to have any effect you are copying data from a zero filled vector a zero filled array.  You only need `let data_array = [0u32; 4096];`. 

It is fine to leave the cursor image supoort not finished for now in this PR, just mentioning this.
For a final implementation we need to copy the data  associated with `resource_id` into a temporary array, witch is sent to the gpu_backend.

Reply:
yes, I agree I can add a todo here to not forget.